evaluation/performance_evaluation.py

```diff
@@ -10,7 +10,8 @@
 
 from eden.util import random_bipartition_iter
 
-from rna_design.rna_synthesizer import RNASynth
+# from rna_design.rna_synthesizer import RNASynth
+from rna_design.rna_synthesizer_n import Initializer
 
 from util.dataset import split_to_train_and_test
 from util.dataset import fit_evaluate
```

Comment:
I would bring the various modules from the util directory to the evaluation directory since they are used only during the evaluation